source/_integrations/alexa_devices.markdown

```diff
@@ -19,7 +19,7 @@ ha_platforms:
   - sensor
   - switch
 ha_integration_type: hub
-ha_quality_scale: silver
+ha_quality_scale: platinum
```

Comment:
Core PR forgot to udpate docs. Related PR: https://github.com/home-assistant/core/pull/152905